onnx/defs/schema.h

```diff
@@ -558,6 +560,14 @@ class OpSchema final {
     return tensor_inference_function_ ? true : false;
   }
 
+  bool is_function() const {
```

Comment:
same here... `contains_function` or `has_function` feels more appropriate to me.